src/gnssro/gnssro_bufr2ioda.py

```diff
@@ -456,5 +526,11 @@ def concat_obs_dict(obs_data, append_obs_data):
         help='only encode bending angle ignore refractivity and profiles',
         action='store_true', required=False)
 
+    optional.add_argument(
+        '--no-tp-drift',
+        help='Disable tangent point drift by using the average lat, lon as tangent '
+             'point for all rays in a profile',
+        action='store_true', required=False)
+
```

Comment:
recommend `--use-average-tangent-point` and the default is `False`

```suggestion
    optional.add_argument(
        '--use-average-tangent-point',
        help='Enable (default) tangent point drift or disable and use the average lat, lon as tangent '
             'point for all rays in a profile',
        action='store_true', required=False)

```